docs/tutorials.rst

```diff
@@ -22,13 +22,11 @@ These are a collection of tutorials we consider essential for any new developer
 #. :doc:`how-to-add-a-new-script-to-the-cea`
 #. :doc:`how-to-test-the-cea`
 #. :doc:`how-to-set-up-nsis`
-#. :doc:`how-to-create-a-new-release`
```

Comment:
I'm not familiar with these two files unfortunately. Why did you decide to remove them?
Are they obsolete?

Reply:
those links are broken, and they are already linked somewhere else: `[docs](https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/master/docs)/[developer](https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/master/docs/developer)/developer-walk-through.rst`

Reply:
ok, perfect. In that case, the PR can be merged in my opinion @lguilhermers